### ISSUE TITLE
zrobot fails to start if a service is created from a template which is not on the master branch

### ISSUE THREAD

**Issue by LeeSmet** [threefoldtech/0-robot#66]
after creating a service from a template on a branch which is not master and restarting the robot, the robot fails to start, complaining that it does not find the template for the service

**Comment by zaibon** [comment]
Indeed, the reason for this is we don't keep the commit hash of the version of the service that are deployed.
One solution to solve this and improve the version management of the template would be to save the commit id of the service.
That would allow the robot to get the precise version of template when restarting.
Next to that, this would also allow to load different version of a same template as long as the version of the template changed.
